runtime/src/sorted_storages.rs

```diff
@@ -1,17 +1,32 @@
 use {
-    crate::accounts_db::SnapshotStorage, log::*, solana_measure::measure::Measure,
-    solana_sdk::clock::Slot, std::ops::Range,
+    crate::accounts_db::SnapshotStorage,
+    log::*,
+    solana_measure::measure::Measure,
+    solana_sdk::clock::Slot,
+    std::ops::Range,
+    std::ops::{Bound, RangeBounds},
 };
 
+/// Provide access to SnapshotStorages sorted by slot
 pub struct SortedStorages<'a> {
+    /// range of slots where storages exist (likely sparse)
     range: Range<Slot>,
+    /// the actual storages. index is (slot - range.start)
     storages: Vec<Option<&'a SnapshotStorage>>,
     slot_count: usize,
     storage_count: usize,
 }
 
 impl<'a> SortedStorages<'a> {
-    pub fn get(&self, slot: Slot) -> Option<&SnapshotStorage> {
+    /// primary method of retreiving (Slot, SnapshotStorage)
+    pub fn get_range_iter<R>(&'a self, range: R) -> SortedStoragesIter<'a>
```

Comment:
What about calling this something like `iter_range()` or `iter_over()` or `iter_over_range()`? I think that would match the std iter naming, and it doesn't feel like iterating is the same as "get"-ing. What do you think?